content/ledger-objects/accountroot.md

```diff
@@ -52,14 +52,15 @@ AccountRoot objects can have the following flag values:
 
 | Flag Name | Hex Value | Decimal Value | Description | Corresponding [AccountSet Flag](reference-transaction-format.html#accountset-flags) |
 |-----------|-----------|---------------|-------------|-------------------------------|
-| lsfPasswordSpent | 0x00010000 | 65536 | Indicates that the account has used its free SetRegularKey transaction. | (None) |
+| lsfPasswordSpent | 0x00010000 | 65536 | The account has used its free SetRegularKey transaction. | (None) |
```

Comment:
If these are sorted numerically by hex value, that should probably be the first column.

Reply:
They are, but the flag name is the more useful column, so I'll re-sort them instead.